build-utils/interpolate-all-results.js

```diff
@@ -1,14 +1,41 @@
 var parseEnvsVersions = require("./parse-envs-versions");
 
-module.exports = function interpolateAllResults(tests, envs) {
+module.exports = function interpolateAllResults(tests, envs, customResults = {}) {
```

Comment:
can you elaborate on what customResults is for?

Reply:
This is to be able to supply build.js with test results for browser/browserVersions that aren't part of this repo.

As the test results aren't in json files, but in an exported object in the data-esXxxx.js files, i couldn't come up with another way to provide builds.js with results for for example a nightly.

Also see the changes I made to the readme, in which I (tried to) explain how this would be used.

BTW: did you see my question about whether the es*/compiler-skeleton.html are still being user? I think they are used when using the `compilers` argument, but it looks to me that the resulting pages aren't hosted anywhere, so might this all be obsolete stuff that could be removed?